package-lock.json

```diff
@@ -8074,51 +7777,6 @@
       "resolved": "https://registry.npmjs.org/tr46/-/tr46-0.0.3.tgz",
       "integrity": "sha1-gYT9NH2snNwYWZLzpmIuFLnZq2o="
     },
-    "node_modules/ts-node": {
```

Comment:
https://github.com/facebook/jest/releases/tag/v29.4.0

https://github.com/facebook/jest/pull/13702
https://github.com/facebook/jest/blob/4bc0e8acaf990e6618a7bed1dca67760c20bb12a/packages/jest-config/package.json
https://github.com/facebook/jest/blob/f91898d3129f813e6a9da57648e5b741de0f82e9/docs/Configuration.md
https://github.com/facebook/jest/blob/0b679b81219bda1ed4fc259af6066e7744310ef0/packages/jest-config/src/readConfigFileAndSetRootDir.ts#L124

Removed ts-node is in https://github.com/kachick/wait-other-jobs/pull/372/files#diff-7ae45ad102eab3b6d7e7896acd08c427a9b25b346470d7bc6507b6481575d519R48, what is this time lag? 🤔 


Removing ts-node is looks not yet supported. 👀 https://github.com/facebook/jest/pull/13742